templates/components/status.html.twig

```diff
@@ -0,0 +1,16 @@
+{% macro status(text, options) %}
+    {% set _color = options.color ?? 'green' %}
+    {% set _animated = options.animated ?? true %}
```

Comment:
does that really work? I always had in mind that this will fail if you pass `false` - as in the next two lines. Or does it only fail if you set `true` as default param and want to pass `false`?

Reply:
Yes it does ;-) at least this is how I am using it. I have attached as screenshot with some "tests".

`??` does check for `isset() && !is_null()`  hence this "shorthand" notation. Therefore you can pass `FALSE` without any problems. [(Stackoverflow)](https://stackoverflow.com/questions/53610622/what-does-double-question-mark-operator-mean-in-php)

```twig
    {% set options = {'color': 'success', 'with_dot': true,'animated': true, 'lite': false, 'extraClass': 'me-3'} %}
    {{ status('Demo 1', options) }}

    {% set options = {'color': 'info', 'with_dot': false,'animated': true, 'lite': true, 'extraClass': 'me-3'} %}
    {{ status('Demo 2', options) }}

    {% set options = {'color': 'orange', 'with_dot': true,'animated': true, 'lite': false, 'extraClass': 'me-3'} %}
    {{ status('Demo 3', options) }}

    {% set options = {'color': 'pink', 'extraClass': 'me-3'} %}
    {{ status('Demo 4', options) }}

    {{ status('Demo 5') }}

    {{ status('Demo 6', false) }}

    {{ status('Demo 7', null) }}
```

![grafik](https://user-images.githubusercontent.com/3634653/176989906-6eeb0ba6-1d46-42fe-a0a9-5a95eada79f5.png)

Reply:
Thanks, and I I will use those for the demo app 👍